supertokens_python/recipe/session/session_class.py

```diff
@@ -112,3 +121,105 @@ async def get_expiry(self, user_context: Union[Dict[str, Any], None] = None) ->
             raise_unauthorised_exception("Session does not exist anymore.")
 
         return session_info.expiry
+
+    async def assert_claims(
+        self,
+        claim_validators: List[SessionClaimValidator],
+        user_context: Union[Dict[str, Any], None] = None,
+    ) -> None:
+        original_session_claim_payload_json = json.dumps(
+            self.get_access_token_payload()
+        )
+
+        new_access_token_payload = self.get_access_token_payload()
+        validation_errors: List[ClaimValidationError] = []
+        for validator in claim_validators:
```

Comment:
This was also changed: we first loop through all validators and update them, then loop through them again to run the validations

Reply:
Just like the node PR, this has been done in a seperate PR 

https://github.com/supertokens/supertokens-python/pull/211